modules/monitoring-configuring-the-cluster-monitoring-stack.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// monitoring/configuring-the-monitoring-stack.adoc
+
+[id='configuring-the-cluster-monitoring-stack-{context}']
+= Configuring the cluster monitoring stack
+
+You can configure the Prometheus Cluster Monitoring stack using ConfigMaps. ConfigMaps configure the Cluster Monitoring Operator, which in turn configures components of the stack.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Start editing the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Put your configuration under `data/config.yaml` as key-value pair `<component_name>:{nbsp}<component_configuration>`:
++
+[source,yaml,subs=quotes]
+----
+data:
+ config.yaml: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+   name: *cluster-monitoring-config*
+   namespace: *openshift-monitoring*
+  data:
+   config.yaml: |
+     *_component_*:
+       *_configuration for the component_*
+----
++
+Substitute `<component>` and `<configuration for the component>` accordingly.
++
+For example, create this ConfigMap to configure a Persistent Volume Claim (PVC) for Prometheus:
++
+[source,yaml,subs=quotes]
+----
+data:
+ config.yaml: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+   name: cluster-monitoring-config
+   namespace: openshift-monitoring
+  data:
+   config.yaml: |
+     *prometheusK8s*:
+       *volumeClaimTemplate:
+        spec:
+          resources:
+            requests:
+              storage: 40Gi*
+----
++
+Here, *prometheusK8s* defines the Prometheus component and the following lines define its configuration.
+
+. Apply the new configuration:
```

Comment:
No need for this step, and the command will return error.
Just save the configmap is OK, the configuration will be applied.

#oc -n openshift-monitoring apply configmap cluster-monitoring-config
Error: required flag(s) "filename" not set